src/Datagrid/DatagridInterface.php

```diff
@@ -104,4 +104,10 @@ public function hasActiveFilters();
      * @return bool
      */
     public function hasDisplayableFilters();
+
+    /**
+     * NEXT_MAJOR: Uncomment getSortParameters and getPaginationParameters.
+     */
+    // public function getSortParameters(FieldDescriptionInterface $fieldDescription): array;
```

Comment:
Can you please add `@method` to the interface instead

Reply:
If I add `@method` to the interface, it provides autocompletion for someone using the interface.

Reply:
That's the reason. You could use autocompletion, so we can introduce new methods to stay BC.

Reply:
Done @core23